docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md

```diff
@@ -41,45 +41,52 @@ Complete the steps in one of the following sections to set up an add-in catalog
+5. On the **App Catalog Site** page, choose **OK** to accept the default option and create a new add-in catalog site.
 
-8. Set the  **Server Resource Quota** to 0 (zero). (The server resource quota is related to throttling poorly performing sandboxed solutions, but you won't be installing any sandboxed solutions on your add-in catalog site.)
+6. On the **Create App Catalog Site Collection** page, specify the title of your App Catalog site.
 
-9. Choose  **OK**.
+7. Specify the **Web Site Address**.
 
-10. To add an add-in to the Add-in Catalog Site, browse to the site you have just created. In the left navigation pane, choose  **Office Add-ins**, and then, to upload an Office Add-in manifest file, choose  **new add-in**.
+8. Specify an **Administrator**.
 
-## Publish an add-in to an add-in catalog
+9. Set the **Server Resource Quota** to 0 (zero). (The server resource quota is related to throttling poorly performing sandboxed solutions, but you won't be installing any sandboxed solutions on your app catalog site.)
 
-To publish an add-in to an add-in catalog, complete the following steps.
+10. Choose **OK**.
 
-1. Browse to the add-in catalog:
+The app catalog is now set up.
 
-    - Open the SharePoint Central Administration main page.
+## Publish an add-in to an app catalog
 
-    - Select  **Add-ins**.
+To publish an add-in to an existing app catalog, complete the following steps.
 
-    - Select  **Manage Add-in Catalog**.
-
-    - Choose the link provided, and then choose  **Office Add-ins** on the left navigation bar.
-
-2. Choose the  **Click to add new item** link.
-
-3. Choose  **Browse**, and then specify the [manifest](../develop/add-in-manifests.md) to upload.
+1. Go to the Microsoft 365 admin center. For information on how to find the admin center, see [About the Microsoft 365 admin center](https://docs.microsoft.com/office365/admin/admin-overview/about-the-admin-center)
+2. On the Microsoft 365 admin center page, expand the list of **Admin centers**, and then choose **SharePoint**.
+    > [!NOTE]
+    > You need to use the Classic SharePoint admin center to set up the catalog. If you are in the new SharePoint admin center, choose **Class SharePoint admin center** in the left pane.
+3. In the left task pane, choose  **apps**.
+4. On the **apps** page, choose **App Catalog**.
+5. Choose **Distribute apps for Office**.
+6. In the **Apps for Office** page, choose **New**.
+7. In the **Add a document** dialog, select the **Choose Files** button.
+8. Locate and specify the [manifest](../develop/add-in-manifests.md) file to upload and choose **Open**.
+9. In the **Add a document** dialog, choose **OK**.
 
     Content and task pane add-ins in this catalog are now available from the  **Office Add-ins** dialog box. To access them, choose **My Add-ins** on the **Insert** tab, and then choose **MY ORGANIZATION**.

```

Comment:
Suggest that you remove indentation from the start of this line (so that it's left-aligned within the section and not indented as part of step # 9).